### PR TITLE
Language/grammar pass on rabbit clustering doc

### PR DESCRIPTION
**Pull request by bording**

### CODE REVIEW

File: nservicebus/rabbitmq/cluster.md [hunk 1]
```diff
@@ -3,13 +3,13 @@ title: Using a RabbitMQ cluster
 summary: Guidelines for using RabbitMQ in a cluster
 ---
 
-The [RabbitMQ Clustering Guide](https://www.rabbitmq.com/clustering.html) recommends not bake in the hostnames or IP addresses of the machines hosting nodes in a RabbitMQ cluster in the client applications and instead use an approach such DNS or load balancing. The same is recommended when using the NServiceBus RabbitMQ transport. For more details see the guide.
+The [RabbitMQ Clustering Guide](https://www.rabbitmq.com/clustering.html) recommends that client applications should not bake in the hostnames or IP addresses of the machines hosting nodes in a RabbitMQ cluster. An approach such as DNS or load balancing should be used instead. This approach is recommended when using the NServiceBus RabbitMQ transport. For more details, see the guide.
```
Comment (SimonCropp) [inline nservicebus/rabbitmq/cluster.md]: "see the guide"  should this be a link?

Comment (SimonCropp) [inline nservicebus/rabbitmq/cluster.md]: "DNS or load balancing"  is it possible to link somewhere for these topics?

Comment (bording) [inline nservicebus/rabbitmq/cluster.md]: > "see the guide" should this be a link?

It would be the same link that's at the start of the paragraph.

Comment (bording) [inline nservicebus/rabbitmq/cluster.md]: > "DNS or load balancing" is it possible to link somewhere for these topics?

Hmm, not sure on that. Their doc just mentions them and basically says "these things are out of scope for us to explain"

I guess I could try and find some generic docs on tcp load balancers or a dynamic dns setup?

Comment (SimonCropp) [inline nservicebus/rabbitmq/cluster.md]: ok dont worry about it. just figured if it was simple we could link to it.

yep and i see the paragraph starts with the guide
